function.go

```diff
@@ -0,0 +1,31 @@
+// Copyright 2021 Roger Chapman and the v8go contributors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+package v8go
+
+// #include "v8go.h"
+import "C"
+import (
+	"unsafe"
+)
+
+// Function is a JavaScript function.
+type Function struct {
+	*Value
+}
+
+// Call this JavaScript function with the given arguments.
+func (fn *Function) Call(args []*Value) (*Value, error) {
```

Comment:
Can we change the `args` to be the `Valuer` interface? Then you can use any Value type. ie Object etc

Reply:
Good catch!

Reply:
Updated